README.md

```diff
@@ -1,28 +1,29 @@
 # Notulen prepublish service
-This service acts on document containers. Provides the following calls:
+This service can be used to both sign and publish documents generated from information provided in GN. 
+To facilitate this it also provides calls to generate a preview of the document as it will be published/signed.
 
-- `/prepublish/agenda/:zittingIdentifier`: builds a publication from the agenda of a zitting present in the database, and returns it without persisting it
-- `/signing/agenda/sign/:kind/:zittingIdentifier`:  persists (if it doesn't exist yet) an agenda publication for :kind and creates a signedResource linked to the agenda.
-- `/signing/agenda/publish/:kind/:zittingIdentifier`:  persists  (if it doesn't exist yet) an agenda publication for :kind and creates a publishedResource linked to the agenda.
+Once a specific document is generated for signing or publication the content is "locked". The prepublish service will always reuse the previously generated document for new signatures or publications. This means that if for example this service is used to sign a decision list, the same generated document will be used for publication even when changes have been made to the linked decisions in the mean time. An exception is the notulen publication, as the notulen publication can remove content it will: TODO verify how it's currently set up.
```

Comment:
should update the TODO after we've verified (and document) how the notulen publication works